WD: Salary survey project

goatlady opened the pull request:
<!-- readthedocs-preview writethedocs-www start -->
----
📚 Documentation preview 📚: https://writethedocs-www--2123.org.readthedocs.build/

<!-- readthedocs-preview writethedocs-www end -->

goatlady committed:
add salary survey project

plaindocs commented on docs/conf/portland/2024/writing-day.rst:
```diff
@@ -164,6 +164,19 @@ Our Writing Day goal is to onboard contributors and improve the Museum API defin
+
+Project Organizer Kay Smoljak, she/her
+
+The WTD Salary Survey has been running annually since 2019, with the goal of identifying appropriate salary ranges and providing a basis for pay negotiations, as well as spark discussion on other topics such as remote work, pay transparency, and job satisfaction.   
```
```suggestion
The WTD Salary Survey has been running annually since 2019, with the goal of identifying appropriate salary ranges and providing a basis for pay negotiations, as well as sparking discussion on other topics such as remote work, pay transparency, and job satisfaction.   
```

plaindocs commented on docs/conf/portland/2024/writing-day.rst:
```diff
@@ -164,6 +164,19 @@ Our Writing Day goal is to onboard contributors and improve the Museum API defin
+
+The WTD Salary Survey has been running annually since 2019, with the goal of identifying appropriate salary ranges and providing a basis for pay negotiations, as well as spark discussion on other topics such as remote work, pay transparency, and job satisfaction.   
+ 
+The format, questions and instructions have evolved each year, based on the submitted responses and feedback from the community, but it's largely been a one-person effort. The 2023 results - which will be available before Writing Day - are drawn from the largest respondent pool yet, and for the first time include structured as well as freeform feedback on the survey itself.
```
Maybe reword `The format, questions and instructions` ?

Do you mean `The format, questions, and instructions` or

plaindocs reviewed (approved):
Looks great

rosewms committed:
Update docs/conf/portland/2024/writing-day.rst

Co-authored-by: Sam <sam@writethedocs.org>

rosewms committed:
Salary Survey edits

Added sponsorship notes to GitBook and noted that Museum API is built by Redocly for educational purposes.

rosewms commented on docs/conf/portland/2024/writing-day.rst:
```diff
@@ -156,14 +156,39 @@ Already familiar with GitBook? We're looking for people like you to brainstorm f
+Write the Docs Documentation Salary Survey
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Project Organizer Kay Smoljak, she/her. Afternoon session.
```
@goatlady and @plaindocs - I edited this for style and format as well as some content reordering. All the content was awesome so my goal was to make it friendlier to folks scanning the Project List.

plaindocs reviewed (approved):
Looks good @rosewms , hit it! 

(there is a bonus typo in the Open API section above)

rosewms commented:
@plaindocs I'll get in the next project PR that I'm about to draft.
